ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/generic-list-console/.content.xml

```diff
@@ -305,7 +305,6 @@
                 sling:resourceType="granite/ui/components/coral/foundation/panel/railpanel"/>
         </rails>
     </jcr:content>
-    <createlistwizard/>
```

Comment:
Was this intended to be removed?

Reply:
Empty nodes are only for ordering purposes, and as that node is not there in the first place (in the best case) this will lead to validation issues with FileVault and is therefore consequently be removed here.